Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -454,6 +454,22 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.8,
 )
 
+class UpgradeWorkflow_JMENano(UpgradeWorkflow):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Nano' in step:
+            stepDict[stepName][k] = merge([{'--customise_commands': '"from PhysicsTools.NanoAOD.custom_jme_cff import PrepJMECustomNanoAOD_MC; PrepJMECustomNanoAOD_MC(process)"'}, stepDict[step][k]])
```

Comment:
It would be preferable to use the actual customization function syntax:
```
--customise PhysicsTools/NanoAOD/custom_jme_cff.PrepJMECustomNanoAOD_MC
```

However, this would require that the functions `PrepJMECustomNanoAOD()`, `PrepJMECustomNanoAOD_MC()`, and `PrepJMECustomNanoAOD_Data()` in https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/python/custom_jme_cff.py return the updated process object, which they currently don't. @nurfikri89

Reply:
Apologies for the late reply. Should I make a PR about this or can @srimanob just make the change in this PR?

Reply:
I think it's find to include the change in this PR, as it is relatively minor.